src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +74,18 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should call componentDidUpdate', () =>{
```

Comment:
Are you testing a built in reactjs method? 
I do not see the relevance of this ticket in relation to what the ticket is supposed to fix.

Reply:
I have renamed it to better explain what it does